### PR TITLE
group dataset with household

### PR DESCRIPTION
**Pull request by eleenkmail**
- display the result of grouping  data frame with household id 
- display the result of grouping data frame with person id

From household grouping: there are many features (columns) that have values just for one member of the household and the other members have values ( 0 or nan or 995 etc) which actually must be the same value as the member with the actual value.

### CODE REVIEW

**Review by zubaidasader — COMMENTED** [review]
Well done.

File: notebooks/grouping.ipynb [hunk 1]
```diff
@@ -0,0 +1,1868 @@
...
+    "# display result\n",
+    "household_grouped"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### There are many features (columns) that have values just for one member of the household and the other members have values ( 0 or nan or 995 etc) which actually must be the same value as the member with the  real value. "
```
Comment (Rama-Has) [inline notebooks/grouping.ipynb:912]: Can u please mention those features names

**Review by Rama-Has — COMMENTED** [review]
Please check "grouping.ipynb" comment

**Review by zubaidasader — APPROVED** [review]

**Comment by mawada-sweis** [comment]
Depen on the result of the grouped dataset with the household column, **we decide to delete all non-participant persons**.

**Pull request closed without merging**